x-pack/plugins/screenshotting/server/browsers/chromium/driver.ts

```diff
@@ -345,8 +345,7 @@ export class HeadlessChromiumDriver {
       return;
     }
 
-    // FIXME: retrieve the client in open() and  pass in the client
-    const client = this.page.client();
+    const client = (this.page as unknown as { _client(): CDPSession })._client();
```

Comment:
There is a better way to do that in TypeScript.

```typescript
// ...

declare module 'puppeteer' {
  interface Page {
    _client(): CDPSession;
  }

  interface Target {
    _targetId: string;
  }
}

// ...

    const client = this.page._client();
    const { _targetId: targetId } = target;
```

Reply:
@vadimkibana That was in the previous version, `15.5.0` doesn't have this private property anymore. I've checked that locally.

Reply:
Done.